lib/rets/client.rb

```diff
@@ -9,11 +9,13 @@ class Client
 
     def initialize(options)
       @options = options
-      @capabilities = nil
       clean_setup
     end
 
     def clean_setup
+      if options.fetch(:login_after_error, false)
```

Comment:
This makes sense to me, but I'm wondering if the default should be `true`? That's how it worked before #136 right?